media-libs/portaudio/portaudio-19.06.00-r1.ebuild

```diff
@@ -51,6 +51,14 @@ src_compile() {
 	fi
 }
 
+multilib_src_install() {
+	emake DESTDIR="${D}" install
+
+	if multilib_is_native_abi; then
+		use examples && dobin bin/.libs/*
```

Comment:
this is a bad idea as it might leak rpaths & co

better have an install target here

Reply:
portaudio's make-based build system doesn't have any, and to use cmake, one must convert the whole ebuild to use it, afaiu.